u2f-gae-demo/war/js/u2fdemo.js

```diff
@@ -28,6 +28,18 @@ function tokenToDom(token) {
   } else {
 	  card.querySelector('.transportsValue').textContent = token.transports;
   }
+  if (token.android_attestation != null && token.android_attestation !== undefined) {    
```

Comment:
I think you could rely on "truthiness," i.e. if (token.android_attestation) { ... }